site/term-manager/tool/src/java/org/sakaiproject/acadtermmanage/tool/util/ReversableComparator.java

```diff
@@ -0,0 +1,30 @@
+package org.sakaiproject.acadtermmanage.tool.util;
+
+import java.util.Comparator;
+
+/**
+ * Reverses the result of a Comparator.
+ * It's a helper class which provides an easy way to reverse the sort order of a list. 
+ * 
+ */
+public class ReversableComparator<T> implements Comparator<T> {
```

Comment:
Does the java 8 lambda reverse suffice?

Reply:
Possibly, but we can probably just do a Collections.reverse too . . .